tools/sysadmin-vm-prep.sh

```
@@ -0,0 +1,138 @@
+fi
+
+# Step 1: Bootstrap Identity
+echo -e "\n${YELLOW}Step 1: Establishing Identity (as $REMOTE_USER)...${NC}"
+echo -e "${YELLOW}Note: You will be prompted for SSH and Sudo passwords for '$REMOTE_USER' now.${NC}"
+
+# Override restrictive ansible.cfg ssh_args that enforce publickey
+export ANSIBLE_SSH_ARGS="-o ControlMaster=auto -o ControlPersist=60s -o PreferredAuthentications=password,publickey"
+
+ansible-playbook -i "$TARGET_HOST," \
+    playbooks/bootstrap-node.yml \
+    $LOCAL_FLAG \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)